checkov/terraform/output/record.py

```diff
@@ -51,13 +54,21 @@ def __str__(self):
         file_details = colored(
             "\tFile: {}:{}\n\n".format(self.file_path, "-".join([str(x) for x in self.file_line_range])),
             "magenta")
-        code_lines = "{}\n".format("".join(
-            [self._code_line_string(line_num, line) for (line_num, line) in self.code_block]))
+        if self.code_block:
+            code_lines = "{}\n".format("".join(
+                [self._code_line_string(line_num, line) for (line_num, line) in self.code_block]))
+        if self.evaluations:
+            for (var_name, evaluations) in self.evaluations.items():
+                for expression in evaluations['expressions']:
+                    evaluation_message = evaluation_message + colored(
+                        f'\tVariable {colored(var_name, "yellow")} evaluated to value {colored(evaluations["value"], "yellow")} '
+                        f'in expression: {colored(evaluations["definition"] + " = ", "yellow")}{colored(expression["definition_expression"], "yellow")}\n',
+                        'white')
         status_message = colored("\t{} for resource: {}\n".format(status, self.resource), status_color)
-        if self.check_result['result'] == CheckResult.FAILED:
-            return check_message + status_message + file_details + code_lines
+        if self.check_result['result'] == CheckResult.FAILED and code_lines:
+            return check_message + status_message + file_details + code_lines + evaluation_message
```

Comment:
is it printing all of the files evaluations or only the ones that are relvant to the check's block?

Reply:
Relevant to the block